client/client.go

```diff
@@ -0,0 +1,145 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/mikloslorinczi/infra-exec/common"
+	"github.com/pkg/errors"
+)
+
+const envFile = "client.env"
+
+var (
+	nodeName string
+	nodeTags string
+	period   int
+)
+
+func main() { // nolint: gocyclo
+
+	status, err := initConfig()
+	if err != nil {
+		fmt.Println(err)
+		printUsage()
+		os.Exit(status)
+	}
+
+	fmt.Printf("\nInfra Client - %v initialized.\nWith the tags: %v\nPolling Infra Server @ %v every %v seconds\n\n", nodeName, nodeTags, common.APIURL, period)
+
+	for running := true; running; {
+		// Calculate next tick
+		nextTick := time.Now().Add(time.Second * time.Duration(period))
```

Comment:
Tickers should be closed in defer blocks at any case.

https://golang.org/pkg/time/#example_NewTicker